android/gradle/libs.versions.toml

```diff
@@ -19,21 +19,20 @@ emergePlugin = "3.1.1"
 emergeSnapshots = "1.1.2"
 composeCompilerExtension = "1.5.3"
```

Comment:
No current action needed here (or soon), but was playing around with https://developer.android.com/develop/ui/compose/compiler the other day, would be good to move to that at some point in the near future, but again, no rush.

Reply:
Oh snap, good mention. I'll put up a diff to move over soon.